src/argilla/monitoring/asgi.py

```diff
@@ -190,14 +256,16 @@ def _log_to_argilla(
         url: str,
         **tags
     ):
-        records = [
-            record
-            for _inputs, _outputs in zip(inputs, outputs)
-            for record in [self._records_mapper(_inputs, _outputs)]
-            if record
-        ]
-
-        if records:
+        records = self._records_mapper(inputs, outputs)
```

Comment:
maybe it makes sense to create a try-except for the records mapper that provides some information about the input-output and that the _records_mapper is causing the error and can be adapted manually.

Reply:
+1